docs/pipelines/includes/get-status-badge.md

```diff
@@ -33,7 +33,7 @@ Now with the badge Markdown in your clipboard, take the following steps in GitHu
 
 4. Notice that the status badge appears in the description of your repository.
 
-To configure anonymous access to badges:
+To configure anonymous access to badges for private repositories:
```

Comment:
```suggestion
To configure anonymous access to badges for private projects:
```
Can you change the wording to project from repositories? The setting that determines whether you can see the toggle is whether your project is public.

Reply:
Thanks, yep, good suggestion!